decentraland/unity-renderer
# fix: messages are mark as seen only once when receiving many in a row

## What does this PR change?

We were spamming mark messages as seen to kernel for each of the messages we retrieved from the history. Now it is requested only once.

## How to test the changes?

1. Navigate through all chat windows: #nearby #channels and direct message
2. Send messages
3. Close chat windows and receive messages
4. Unseen messages counter should increase
5. Open the chat windows, the unseen messages for that specific chat should be reset

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
